src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -137,11 +136,11 @@ private async Task<Build> GetRootBuildAsync()
                     return null;
                 }
                 Console.WriteLine($"Looking up builds of {_options.RepoUri}@{_options.Commit}");
-                IList <Build> builds = await remote.GetBuildsAsync(_options.RepoUri, _options.Commit);
+                IEnumerable<Build> builds = await remote.GetBuildsAsync(_options.RepoUri, _options.Commit);
                 // If more than one is available, print them with their IDs.
-                if (builds.Count > 1)
+                if (builds.Count() > 1)
                 {
-                    Console.WriteLine($"There were {builds.Count} potential root builds.  Please select one and pass it with --id");
+                    Console.WriteLine($"There were {builds.Count()} potential root builds.  Please select one and pass it with --id");
```

Comment:
>builds.Count() [](start = 52, length = 14)

Store the count in a variable when called in line 141 so we don't have to go through the whole collection again.